### ISSUE TITLE
Species selector, each group needs "select all", "deselect all", maybe "invert all"

### ISSUE THREAD

**Issue by CGDogan** [btovis/btovis.github.io#147]

**Comment by Hex27** [comment]
The group selector operates independently of which species is currently ticked, so this feature is technically already present

If the group is not selected at the top, it will be filtered away

**Comment by CGDogan** [comment]
One sec, let's say the user would like to deselect all birds except one. If he disables from top, he won't be able to select that one?

**Comment by Hex27** [comment]
yea, it's not easy to mix groups together if the user is seeking to have 1 bird with everything else

**Comment by CGDogan** [comment]
Other than "select deselect maybe invert" buttons another clean way to fix this might be that the top buttons don't hide the checkboxes, but unchecks all of them, so if the user checks one of them, the top button is active again

**Comment by Hex27** [comment]
That might break the use case where the user is only interested in birds, because now you'd have to scroll past all the unchecked groups

Is a checkbox beside each group name better for this?

**Comment by CGDogan** [comment]
> Is a checkbox beside each group name better for this?

I find the top buttons very useful but this too would be useful.

I just found about this in HTML:

<details>
  <summary>Birds</summary>
  <p>Bird1.</p>
  <p>Bird2.</p>
  <p>Bird3.</p>
</details>

which is
```
<details>
  <summary>Birds</summary>
  <p>Bird1.</p>
  <p>Bird2.</p>
  <p>Bird3.</p>
</details>
```

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_details

which is like accordion but it takes less attention so we can use this for all groups and when everything's unticked in a group, we can hide them under this accordion.

So what I suggest (and feel free to change this) is adding a checkbox to each group that is in sync with the top buttons, and when either the checkbox is unchecked, or the top button is deactivated, the species are hidden under the "accordion" I found.

The default should be keeping all species hidden under "accordion", and when the user clicks on the group checkbox, or the "accordion" triangle, the code can expand it
